cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/HourlyCloudPricingDTO.java

```diff
@@ -0,0 +1,70 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+public class HourlyCloudPricingDTO extends DTO
+{
+    @NotNull
+    private BigDecimal upfront;
+
+    @NotNull
+    private BigDecimal price;
+
+    public HourlyCloudPricingDTO( @NotNull BigDecimal upfront, @NotNull BigDecimal price )
+    {
+        this.upfront = upfront;
+        this.price = price;
+    }
+
+    public HourlyCloudPricingDTO( @NotNull BigDecimal price )
+    {
+        this( new BigDecimal( "0.00" ), price );
+    }
+
+    public HourlyCloudPricingDTO()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "upfront" is marked "javax.validation.constraints.NotNull" but is not initialized in this constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2637)